middlewares/hacker.middleware.js

```diff
@@ -47,13 +47,15 @@ function parseHacker(req, res, next) {
         needsBus: req.body.needsBus,
         application: req.body.application,
     };
+    req.body.token = req.body.authorization;
```

Comment:
Do we need this here?

Reply:
(I.e. do we ever use it)

Reply:
for consistency with resetPassword I move it into token.
I can move it into req.body.confirmAccountToken or something as well